### PR TITLE
Adds a function to utilize the aws-sdk marshall method for parsing JSON

### PR DESCRIPTION
**Pull request by dakota002**
### We really appreciate your commitment to improving Architect

To maintain a high standard of quality in our releases, before merging every pull request we ask that you've completed the following:

- [x] Forked the repo and created your branch from `master`
- [x] Made sure tests pass (run `npm it` from the repo root)
- [ ] Expanded test coverage related to your changes:
  - [ ] Added and/or updated unit tests (if appropriate)
  - [ ] Added and/or updated integration tests (if appropriate)
- [ ] Updated relevant documentation:
  - [x] Internal to this repo (e.g. `readme.md`, help docs, inline docs & comments, etc.)
  - [ ] [Architect docs (arc.codes)](https://github.com/architect/arc.codes)
- [x] Summarized your changes in `changelog.md`
- [ ] Linked to any related issues, PRs, etc. below that may relate to, consume, or necessitate these changes

Please also be sure to completed the CLA (if you haven't already).

Learn more about [contributing to Architect here](https://arc.codes/intro/community).

Thanks again!

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -133,7 +135,7 @@ module.exports = {
               message: 'Which kind of Dynamo Stream event do you want to invoke?',
               choices: [ 'INSERT', 'MODIFY', 'REMOVE' ]
             })
-            payload = mock.tablesStreams(eventName)
+            payload = mock.tablesStreams(eventName, marshallJson(mocks[pragma][name][eventName]))
```
Comment (ryanblock) [inline src/index.js]: I believe at this moment we cannot guarantee that `mocks[pragma][name][mockName]` exists. So this probably needs to be optionally chained and generally better designed to accommodate cases where data is not present.

File: src/index.js [hunk 2]
```diff
@@ -5,6 +5,8 @@ let { prompt } = require('enquirer')
 let colors = require('ansi-colors')
 let update = updater('Invoker')
 let mock = require('./event-mocks')
+const { marshall } = require('@aws-sdk/util-dynamodb')
```
Comment (ryanblock) [inline src/index.js]: `let` → `const` pls!

File: src/event-mocks.js [hunk 3]
```diff
@@ -51,8 +51,8 @@ function tablesStreams (eventName) {
         dynamodb: {
           ApproximateCreationDateTime: new Date() / 1000,
           // Keys + NewImage are in DynamoDB JSON, perhaps we can mock transform that later
-          Keys: { NOT_MOCKED: true },
-          NewImage: { NOT_MOCKED: true },
+          Keys: dynamoData.Keys,
+          NewImage: dynamoData.NewImage,
```
Comment (ryanblock) [inline src/event-mocks.js]: Probably for these we should fall back to `{ NOT_MOCKED: true }` if the data is not present.

**Comment by ryanblock** [comment]
Thank you for your patience, @dakota002!

**Comment by dakota002** [comment]
Apologies for the delay, please let me know if there is anything I need to update for this

**Pull request merged**